app/controllers/trainer/dashboard_controller.rb

```diff
@@ -0,0 +1,19 @@
+module Trainer
+  class DashboardController < BaseController
+
+    def index
+      results = User.search_users(params[:search])
+      if params[:search].blank?
+        @users = User.includes(profile: [:language]).with_any_role({ name: :user, resource: current_user.organization }, { name: :admin, resource: current_user.organization })
+      else
+        @users = results & User.includes(profile: [:language]).with_any_role({ name: :user, resource: current_user.organization }, { name: :trainer, resource: current_user.organization })
```

Comment:
I just noticed that the ```name:``` of the second arg for ```with_any_role``` on this line and line 7 are different (admin on one, trainer on another). Just want to make sure that was intentional, cause that'd be a pain to track down.